ccd-definition/AuthorisationCaseField/CareSupervision/la-managing.json

```diff
@@ -1297,7 +1304,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "manageDocumentsCourtBundleNoHearingsLabel",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json